src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -0,0 +1,76 @@
+from confluent_kafka import Producer
+from django.conf import settings
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, region_silo_endpoint
+from sentry.api.permissions import SuperuserPermission
+from sentry.utils import json
+from sentry.utils.kafka_config import get_kafka_producer_cluster_options
+
+
+class BasicEventSerializer(serializers.Serializer):
+    event_id = serializers.CharField()
```

Comment:
is this something we want to generate client side or server side? my understanding was we'd save the event during ingest and then use that ID

Reply:
I was planning on having it be client side, similar to how our SDKs work today.

Reply:
ah ok. will update on my end then